pkg/distributor/distributor.go

```diff
@@ -322,6 +332,12 @@ func New(cfg Config, clientConfig ingester_client.Config, limits *validation.Ove
 	}, func() float64 {
 		return float64(d.inflightPushRequests.Load())
 	})
+	promauto.With(reg).NewGaugeFunc(prometheus.GaugeOpts{
+		Name: "cortex_distributor_inflight_push_requests_size",
```

Comment:
I would match the config option name.

```suggestion
		Name: "cortex_distributor_inflight_push_requests_bytes",
```

Reply:
changed in https://github.com/grafana/mimir/pull/2413/commits/ae16fcdd91025f7bfa0d8e2faa730fcc1c48725a